app/stores/ada/LedgerConnectStore.js

```diff
@@ -53,6 +60,7 @@ export default class LedgerConnectStore
   // =================== VIEW RELATED =================== //
   @observable progressInfo: ProgressInfo;
   @observable derivationIndex: number = HARD_DERIVATION_START + 0; // assume single account
+  @observable purpose: number = WalletTypePurpose.BIP44;
```

Comment:
This needs to have a new UI dialog to choose whether you want BIP44 mode or CIP1852 mode. I already have the UI dialog itself made, but haven't exposed it in the flow yet.